File: xposed-api-stubs/src/main/java/de/robv/android/xposed/IXposedHookInitPackageResources.java
package de.robv.android.xposed;

import de.robv.android.xposed.callbacks.XC_InitPackageResources;

public interface IXposedHookInitPackageResources extends IXposedMod {
    void handleInitPackageResources(XC_InitPackageResources.InitPackageResourcesParam resparam) throws Throwable;
}

File: app/src/main/java/de/robv/android/xposed/IXposedHookInitPackageResources.kt
package de.robv.android.xposed

import de.robv.android.xposed.callbacks.XC_InitPackageResources

interface IXposedHookInitPackageResources : IXposedMod {
    @Throws(Throwable::class)
    fun handleInitPackageResources(resparam: XC_InitPackageResources.InitPackageResourcesParam?)
}
